admin_frontend/src/config.rs

```diff
@@ -2,21 +2,26 @@ use tracing::warn;
 
 #[derive(Debug, Clone)]
 pub struct Config {
+  pub host: String,
+  pub port: u16,
   pub redis_url: String,
   pub gotrue_url: String,
   pub appflowy_cloud_url: String,
 }
 
 impl Config {
-  pub fn from_env() -> Self {
-    Config {
+  pub fn from_env() ->  Result<Config, anyhow::Error> {
+    let cfg = Config {
+      host: get_or_default("APPFLOWY_ADMIN_HOST", "0.0.0.0"),
+      port: get_or_default("APPFLOWY_ADMIN_PORT", "3000").parse()?,
```

Comment:
for consistency, can we use `ADMIN_FRONTEND` as the prefix instead of `APPFLOWY_ADMIN`?

Reply:
Good idea.